src/sys/DISK.ASM

```diff
@@ -29,11 +29,11 @@ verify:                                         ; Verify that a file exists
     mov si, iface_disk_search
     call comment                                ; Comment message
 
-    push 0x3800
+    push 0x3400
```

Comment:
The start of all directories is now segment `0x3400`, as only 16K is allocated to the FAT. Bootloader still loads root into `0x3800` so as to allow loading in `/HOME` on initialization.